packages/lesswrong/components/comments/CommentsNode.jsx

```diff
@@ -121,6 +121,7 @@ class CommentsNode extends Component {
       return true;
     
     return (
+      this.isTruncated() &&
       currentUser?.beta &&
       lowKarmaOrCondensed &&
       !(mostRecent && condensed) &&
```

Comment:
When I do algebraic simplification on this line, I get
```suggestion
      (!condensed || lastCommentId === comment._id)
```
That doesn't make a whole lot of sense, but if it's wrong, it's at least stating the wrong thing more directly than it was before.

Reply:
I guess the last two lines simplify to
```
  (condensed
    ? lastCommentId === comment._id
    : comment.baseScore < 10)
```